style.css

```diff
@@ -0,0 +1,9 @@
+@media only screen and (max-width: 600px) {
+  section {
+    background-color: lightblue;
+  }
+}
+
+#navbar {
+  position: fixed;
+}
```

Comment:
Kindly I recommend you give this a margin-bottom.

Reply:
- [ ] When you give a code review, be polite and explain why you ask to add the margin. I think it would be better to use margin-bottom to be your design much the template design you chose :smiley_cat: